dao/src/main/scala/za/co/absa/enceladus/dao/MenasRestDAO.scala

```diff
@@ -18,23 +18,22 @@ package za.co.absa.enceladus.dao
 import java.util.UUID
 
 import com.typesafe.config.ConfigFactory
-import org.apache.commons.httpclient.HttpStatus
 import org.apache.http.client.methods.{CloseableHttpResponse, HttpPost}
 import org.apache.http.entity.{ContentType, StringEntity}
 import org.apache.http.impl.client.HttpClients
 import org.apache.log4j.LogManager
 import za.co.absa.atum.model._
 import za.co.absa.atum.utils.ControlUtils
-import za.co.absa.enceladus.dao.EnceladusRestDAO.{csrfToken, sessionCookie}
+import za.co.absa.enceladus.dao.EnceladusRestDAO.{csrfToken, sessionCookie, userName}
```

Comment:
userName is not being used here. Plus if you want to do a full clean up, the ControlUtils aren't used as well.